### ISSUE TITLE
In Android, Receipt Validation Stuck With No Success or Fail Callback

### ISSUE THREAD

**Issue by okanace** [AppsFlyerSDK/appsflyer-unity-plugin#182]
As i mention in the title receipt validation not working in android devices. Weirdly enough if i delete device data, only in first session receipt validation works without a problem. According to the Appsflyer logs, the verification request is blocked by another task. I am posting the logs below.

No callback case:
2022-09-27 11:41:39.293 20354-20643/? D/AppsFlyer_6.8.2: QUEUE: new task added: PURCHASE_VALIDATE-10
2022-09-27 11:41:39.293 20354-20643/? D/AppsFlyer_6.8.2: QUEUE: new task was blocked: SDK_SERVICES-11

Success case (Only in first session): 
2022-09-26 19:30:31.741 1508-2452/? D/AppsFlyer_6.8.2: QUEUE: new task added: PURCHASE_VALIDATE-11
2022-09-26 19:30:31.741 1508-2452/? D/AppsFlyer_6.8.2: QUEUE: new task added: SDK_SERVICES-12
2022-09-26 19:30:31.741 1508-2453/? D/AppsFlyer_6.8.2: QUEUE: starting task execution: PURCHASE_VALIDATE-11

iOS works without problem. Sdk version: 6.8.3

**Comment by pazlavi** [comment]
Hi @okanace, 
Thank you for reaching out to us.

We know this bug in the native Android SDK, and we'll fix it in the upcoming release.
In the meanwhile, you can downgrade to an SDK version **lower** than 6.7.0

**Comment by den-bel** [comment]
@pazlavi Hi!
Has this bug been already fixed?

**Comment by zenith-jacob** [comment]
Nope, it hasn't

**Comment by den-bel** [comment]
@pazlavi  @af-margot What about this bug? Fixed in 6.8.5?
